validation/validate_dc2_run2.1i_object_table.ipynb

```diff
@@ -224,6 +247,134 @@
+    "                                                    filters=[cosmoDC2_subsample] + mag_limit_filter))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'Loaded {len(cosmoDC2)} objects from \"cosmocDC2\" with a subsampling of {sampling_factor}x')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### HSC XMM\n",
+    "from HSC PDR1."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hsc_cat = GCRCatalogs.load_catalog('hsc-pdr1-xmm',\n",
+    "                                   config_overwrite={'base_dir': '/global/cscratch1/sd/desc/hsc',\n",
+    "                                                    'use_cache': False})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Establish some basic cuts intended to yield a galaxy sample with reasonable flux measurements.\n",
+    "basic_cuts = [\n",
+    "    GCRQuery('clean'), # The source has no flagged pixels (interpolated, saturated, edge, clipped...) \n",
+    "                       # and was not skipped by the deblender\n",
+    "    GCRQuery('xy_flag == 0'),                                      # Flag for bad centroid measurement\n",
+    "]\n",
+    "\n",
+    "# Define cuts on galaxy properties: currently we simply make a sample limited at i<25.\n",
+    "# This implicitly requires the magnitude to be finite.\n",
+    "max_mag_i = 25\n",
+    "properties_cuts = [\n",
+    "    GCRQuery(f'mag_i_cModel < {max_mag_i}'),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hsc_filters = ['g', 'r', 'i', 'z']\n",
+    "hsc_columns = ['ra', 'dec']\n",
+    "hsc_columns += [f'mag_{f}' for f in hsc_filters]\n",
+    "hsc_columns += [f'magerr_{f}' for f in hsc_filters]\n",
+    "hsc_columns += [f'mag_{f}_cModel' for f in hsc_filters]\n",
+    "hsc_columns += [f'magerr_{f}_cModel' for f in hsc_filters]\n",
+    "# hsc_columns += [f'Ixx_{f}' for f in hsc_filters]\n",
+    "# hsc_columns += [f'Ixy_{f}' for f in hsc_filters]\n",
+    "# hsc_columns += [f'Iyy_{f}' for f in hsc_filters]\n",
+    "# hsc_columns += [f'psf_fwhm_{f}' for f in hsc_filters]\n",
+    "hsc_columns += ['good', 'extendedness', 'blendedness']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hsc = pd.DataFrame(hsc_cat.get_quantities(hsc_columns,\n",
+    "                                          filters=basic_cuts + properties_cuts))\n",
+    "\n",
+    "# We could of course done this as part of a GCR query.\n",
+    "# But using a query in the DataFrame is how we did the `stars`, `galaxies` selection above, so we'll do it the same way here.\n",
```

Comment:
Where did we do this above?

Reply:
I think we do this below in cell 32.

Reply:
Ah, it used to be above.  I've removed any mention of relative direction.